java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/IPv6AddressTest.java

```diff
@@ -436,6 +436,85 @@ public void testOfExceptions() throws Exception {
         }
     }
 
+    @Test
+    public void testReservedIp() {
+        IPv6Address addr;
+        /* fec0::/10 Reserved by IETF [RFC3879]. Deprecated by
+         * [RFC3879] in September 2004. Formerly a Site-Local scoped
+         * address prefix.
+         */
+        addr = IPv6Address.of("fec0::10");
```

Comment:
would be good to also test, for each of the rules, that the first address that lies outside of the reserved range is correctly reported as not-reserved.

Reply:
I have added a few more tests.